about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>About</title>
+  <LINK REL=StyleSheet HREF="stylesheets/normalize.css" TYPE="text/css" MEDIA=screen>
```

Comment:
html elements should **not** be capitalized.  You should also not capitalize attributes either.